### DISCUSSION TITLE
Poster Access

### DISCUSSION THREAD

**Discussion by skatenroll** [meisnate12/Plex-Meta-Manager-Configs#273] [Q&A]
Hello! I would love to get access to your posters but the link listed seems to be down. Is there any other way I could grab some of them? I would be very willing to donate and even pay to have some more designed - my girlfriend, too, hates that I discovered plex collections... so I feel that for sure.

**Comment by meisnate12** [comment]
So couple of things

1. I highly recommend joining the discord you'll get much better responses 

2. Which link seems to not be working?

**Reply by skatenroll** [reply]
Great, I'll try and join the discord! The "yozora" file with the link: https://mega.nz/folder/RcE1VIZQ#WTwcOGwETnWZEjafLsIFHA 
Are you the one that posted this? Sorry I've been on this site for months and the layout still baffles me.

**Reply by skatenroll** [reply]
Also, my girlfriend also hates plex collections so I know the feeling... hah
